packages/gridjs/src/view/table/messageRow.tsx

```diff
@@ -15,10 +15,11 @@ export interface MessageRowProps extends BaseProps {
 export class MessageRow extends BaseComponent<MessageRowProps, {}> {
   render() {
     return (
-      <TR>
+      <TR stylingRow={true}>
```

Comment:
I like this approach but I don't think `stylingRow` is the best attribute name that can be used here. Can we change it to something a bit more self-describing?

Reply:
i think so. suggest please

Reply:
`messageRow`?